### ISSUE TITLE
Hey, could anyone help me with the encoding error but it gives the error in kable() function

### ISSUE THREAD

**Issue by PriyaShaji** [tidyverse/rvest#234]
Hey, could anyone help me with the same error but it gives the error in kable() function

<img width="904" alt="Screen Shot 2019-03-23 at 11 02 56 PM" src="https://user-images.githubusercontent.com/35895465/54874344-ddfdc980-4dbf-11e9-943f-891a16902077.png">

The error is with the tweet table.
With LinkedIn table, the code works just fine and gives the output for it.

_Originally posted by @PriyaShaji in https://github.com/tidyverse/rvest/issues/203#issuecomment-475923930_

**Comment by PriyaShaji** [comment]
The 3,4,10 column of tweet looks repectively:
<img width="1071" alt="Screen Shot 2019-03-23 at 11 16 19 PM" src="https://user-images.githubusercontent.com/35895465/54874440-af80ee00-4dc1-11e9-8efa-5130331549f3.png">

<img width="269" alt="Screen Shot 2019-03-23 at 11 16 54 PM" src="https://user-images.githubusercontent.com/35895465/54874444-ccb5bc80-4dc1-11e9-8b4c-106ac59bac9b.png">

<img width="499" alt="Screen Shot 2019-03-23 at 11 17 47 PM" src="https://user-images.githubusercontent.com/35895465/54874445-e2c37d00-4dc1-11e9-81dc-f8501df503d2.png">

**Comment by hadley** [comment]
This sort of question is a better fit for <https://community.rstudio.com>. Do you mind asking it over there? (You might want to read <https://www.tidyverse.org/help/> first to maximise your chances of getting a good answer)

**Comment by PriyaShaji** [comment]
Hey, I solved the issue by using a guess_encoding() function. I never came across this function until now. Proved to be very useful. It gave two columns with few rows. The first column consisted of all the names of the encodings that are present in the input file, and the second column gave the percentage of those respective encodings used in the file. So while reading the file as read_csv(), I used the parameter encoding='' (the name of the encoding which had the highest percentage value or the name of the encoding which was used the maximum in the file). It corrected the error.

Thank you all for the suggestions. I appreciate it!
